lms/djangoapps/branding/__init__.py

```diff
@@ -17,12 +17,16 @@
 from openedx.core.djangoapps.content.course_overviews.models import CourseOverview
 
 
-def get_visible_courses():
+def get_visible_courses(org=None):
     """
     Return the set of CourseDescriptors that should be visible in this branded instance
     """
-    filtered_by_org = microsite.get_value('course_org_filter')
-    courses = CourseOverview.get_all_courses(org=filtered_by_org)
+    microsite_org = microsite.get_value('course_org_filter')
+
+    orgs = [org, microsite_org]
+    orgs = [org for org in orgs if org is not None]
+
+    courses = CourseOverview.get_all_courses(orgs=orgs)
```

Comment:
i'm probably missing something, but instead of querying against the union of the orgs passed, could we instead query against the intersection?  Also, now that get_all_courses takes org parameters, is the filtering (starting on line 42 below) actually necessary anymore?